templates/custom/nodes/node--hero-image.html.twig

```diff
@@ -72,7 +72,5 @@
 #}
 
 <figure class="hero">
-  <div class="image-wrapper" data-ratio="4:1">
-    {{ content }}
-  </div>
+  {{ content }}
```

Comment:
Kan je niet beter detecten of de content empty is?

Reply:
Fixed.